lusha_exercise/utils/geographic_utils.py

```
@@ -0,0 +1,61 @@
+import countryinfo
+import pycountry
+from pycountry_convert.convert_continent_code_to_continent_name import convert_continent_code_to_continent_name
+from pycountry_convert.convert_country_alpha2_to_continent_code import country_alpha2_to_continent_code
+
+SPECIAL_COUNTIES_WHITE_LIST = {'netherlands antilles': 'Europe'}
+
+
+def get_region_name(country_name: str):
+    """
+    Return the country's region. For example, for Spain, will return Europe.
+
+    :return: the field that holds the current company side.
+    :rtype: str
+    """
+    try:
+        country_name = country_name.strip()
+        country = pycountry.countries.get(name=country_name) or pycountry.countries.get(
+            official_name=country_name) or pycountry.historic_countries.get(
+            name=country_name)
+        if country is not None:
+            return convert_continent_code_to_continent_name(country_alpha2_to_continent_code(country.alpha_2))
+        try:
+            return countryinfo.countryinfo.CountryInfo(country_name=country_name).region()
+        except KeyError:
+            country = _find_country_by_iterate_all_relevant_countries(country, country_name)
+            return convert_continent_code_to_continent_name(
+                country_alpha2_to_continent_code(country.alpha_2))
+    except Exception as error:
```

It's a bad practice to catch all Exceptions, (also, if you're not using `error` then you don't need to declare it).